packages/client-core/src/components/Settings/ShareSpaceScreen.tsx

```diff
@@ -55,9 +55,10 @@ const ShareSpaceScreen: React.FC<ShareSpaceScreenProps> = () => {
             Copy Direct Link
           </TextButton>
 
-          <TextButton onClick={() => copyLinkToClipboard(questLink)} className="w-full" fade={`dark`}>
+          {/** commenting out in case we need to bring this back */}
```

Comment:
I dont think we need this comment. The commented out code already means what this line is saying.